shell/app/common/components/base-list/list-item.tsx

```diff
@@ -81,191 +83,203 @@ const ListItem = (props: ERDA_LIST.ItemProps) => {
   const curLogo = logoURL || defaultLogo;
   return (
     <div
-      className={`erda-base-list-item cursor-pointer rounded-sm`}
+      className={`erda-base-list-item cursor-pointer`}
       {...itemProps}
       style={backgroundImg || defaultBgImg ? { backgroundImage: `url(${getImg(backgroundImg || defaultBgImg)})` } : {}}
     >
-      <div className="flex">
-        {mainState ? <Badge onlyDot {...mainState} /> : null}
-        {curLogo ? (
-          isString(curLogo) ? (
-            <div className="item-prefix-img flex items-center">{getLogo(curLogo, logoCircle)}</div>
-          ) : (
-            <div className="item-prefix-img flex items-center">{curLogo}</div>
-          )
-        ) : null}
-        {icon ? (
-          <div className="item-prefix-img flex items-center">
-            <ErdaIcon disableCurrent type={icon} size={28} />
-          </div>
-        ) : null}
-        <div className="flex flex-1 overflow-hidden">
-          <div className="flex-1 flex flex-col justify-center overflow-hidden">
-            <div className="body-title">
-              {titlePrefixIcon && (
-                <Tooltip title={titlePrefixIconTip}>
-                  <ErdaIcon type={titlePrefixIcon} className="title-icon mr-2" />
-                </Tooltip>
-              )}
-              <Ellipsis className=" title-text text-base text-default hover:text-purple-deep" title={title} />
-              {titleSummary ? (
-                <span className="inline-block ml-1 bg-default-1 px-1.5 rounded-lg text-default-8 text-xs leading-5">
-                  {titleSummary}
-                </span>
-              ) : null}
-              {titleSuffixIcon && (
-                <Tooltip title={titleSuffixIconTip}>
-                  <ErdaIcon type={titleSuffixIcon} className="title-icon ml-2" />
-                </Tooltip>
-              )}
-              {map(titleState, (item, idx) => (
-                <Badge key={idx} showDot={false} size="small" {...item} className="ml-2" />
-              ))}
-              {tags ? <Tags labels={tags} containerClassName="ml-2" /> : null}
+      <div className="erda-base-list-item-container">
+        <div className="flex">
+          {onSelectChange ? (
+            <Checkbox
+              className="flex items-center mr-2"
+              disabled={!selectable}
+              checked={selected}
+              onClick={(e) => e.stopPropagation()}
+              onChange={(e) => onSelectChange(e.target.checked)}
+            />
+          ) : null}
+          {mainState ? <Badge onlyDot {...mainState} /> : null}
+          {curLogo ? (
+            isString(curLogo) ? (
+              <div className="item-prefix-img flex items-center">{getLogo(curLogo, logoCircle)}</div>
+            ) : (
+              <div className="item-prefix-img flex items-center">{curLogo}</div>
```

Comment:
merge to one line ?`isString(curLogo) ? getLogo(curLogo, logoCircle) : curLogo `